sklearn/decomposition/kernel_pca.py

```diff
@@ -280,6 +302,8 @@ def fit_transform(self, X, y=None, **params):
         """
         self.fit(X, **params)
 
+        # Transform X
+        # (shortcut since we transform the same X that was used to fit)
         X_transformed = self.alphas_ * np.sqrt(self.lambdas_)
```

Comment:
This line is pretty self explanatory, the comment above is not needed.

Reply:
More specifically, this is being transformed because `_fit_inverse_transform` expects this scaling.

Reply:
see my other comment above, I propose to write 

```python
# no need to use the kernel to transform X, use shortcut expression
X_transformed = self.alphas_ * np.sqrt(self.lambdas_)
```